assets/css/dup/free_text.scss

```diff
@@ -95,6 +109,13 @@
   }
 }
 
+.free-text__text-pill__text {
+  display: inline-block;
+  font-family: Helvetica, sans-serif;
+  font-weight: 700;
+  transform: translateY(6px);
```

Comment:
I'd bump this up a bit -- to me it doesn't look centered right now (e.g. look at http://localhost:4000/screen/DUP-SullivanSquare/0).